browser/src/react-redux/App/Nav.react.js

```diff
@@ -1,22 +1,33 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Logout from './Logout/Logout.React';
 
-const User = () => {
-  return (
-    <nav>
-      <ul>
+const User = props => (
+  <nav>
+    <ul>
+      <li>
+        <Link to="/">Home</Link>
+      </li>
+      {!props.user ? (
+        <>
+          <li>
+            <Link to="/signup">Sign Up</Link>
+          </li>
+          <li>
+            <Link to="/login">Login</Link>
+          </li>
+          {' '}
+        </>
+      ) : (
         <li>
```

Comment:
Let's go ahead and hide the logged out link when the user is not logged in.

Reply:
This was already done.  Were you seeing logout button when not signed in?

Reply:
My mistake! This looks good.